development.md

```diff
@@ -0,0 +1,78 @@
+
+You'll want to [install minikube](https://minikube.sigs.k8s.io/docs/start/) and setup your local minikube instance with your container manager of choice. (Docker, Hyperkit, Hyper-V, KVM, Parallels, Podman, VirtualBox, or VMware Fusion/Workstation.)
+
+After installing minikube, run:
+
+```sh
+#!/bin/sh
+
+minikube config set memory 10240
+minikube config set cpus 4
+
+minikube stop
+
+minikube delete
+
+minikube start --addons=dashboard --addons=ingress --addons=olm
+
+kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml
+
+# The below command will fail if the Tackle CRD is not yet established. This may take some time.
+cat << EOF | kubectl apply -f -
+kind: Tackle
+apiVersion: tackle.konveyor.io/v1alpha1
+metadata:
+  name: tackle
+  namespace: konveyor-tackle
+spec:
+EOF
+
+```
+
+Once you've installed Node.js (which includes the popular `npm` package manager) and minikube, open a Terminal and run the following:
+
+```sh
+git clone https://github.com/konveyor/tackle2-ui
+cd tackle2-ui
+npm ci
+npm run start:dev
+```
+
+You should now have a running server! Visit [localhost:9000](http://localhost:4000) in your browser.
+
+When you're ready to stop your local server, type <kbd>Ctrl</kbd>+<kbd>C</kbd> in your terminal window.
+
+Note that `npm ci` is a step that should typically only need to be run once each time you pull the latest for a branch.
+
+- `npm ci` does a clean install of dependencies, without updating the `package-lock.json` file
+
+## Site structure
+
+This site was originally a Ruby on Rails web application. Some time later it was converted into a static site powered by [Jekyll](https://jekyllrb.com/). A few years after that it was migrated to [Nanoc](https://nanoc.app/), another Ruby static site generator.
+
+Today it's a dynamic Node.js webserver powered by Express, using [middleware](../middleware/README.md) to support proper HTTP redirects, language header detection, and dynamic content generation to support the various flavors of GitHub's product documentation, like GitHub.com and GitHub Enterprise Server.
+
+The tooling for this site has changed over the years, but many of the tried-and-true authoring conventions of the original Jekyll site have been preserved:
+
+- Content is written in Markdown files, which live in the `content` directory.
+- Content can use the [Liquid templating language](liquid-helpers.md).
+- Files in the `data` directory are available to templates via the `{% data %}` tag.
+- Markdown files can contain [frontmatter](https://jekyllrb.com/docs/front-matter).
+- The [`redirect_from`](https://github.com/jekyll/jekyll-redirect-from) Jekyll plugin behavior is supported.
```

Comment:
Was this historical information provided from contributors of tackle (v1) ?
I'm asking because that's sounds really strange.
Anyway I wonder if we need to keep such information and how it helps the current project.
For instance there is no trace of Jekyll anywhere but I might be missing something.